constants.py

```diff
@@ -12,3 +12,5 @@
     os.makedirs(DATA_FOLDER + "store/listings/in progress/")
     os.makedirs(DATA_FOLDER + "store/listings/trade receipts/")
     os.makedirs(DATA_FOLDER + "store/media/")
+    os.makedirs(DATA_FOLDER + "purchases/in progress/")
```

Comment:
Probably no spaces in folder names is safer especially for non-Windows machines.